### PR TITLE
hatari 2.1.0

### PR DESCRIPTION
**Pull request by btuduri**
A new version (2.1.0) of hatari but the build failed on the el capitan.
This issue was discovered after the compilation but the build bot.

### CODE REVIEW

**Comment by fxcoudert** [comment]
For what it's worth, the original El Capitan build error is:

```
18:40:45 /tmp/hatari-20180219-73604-1jll9cp/hatari-2.1.0/src/gui-osx/PrefsController.m:679:44: error: use of undeclared identifier 'NSAlertStyleInformational'
18:40:45             applyChanges = [NSApp myAlerte:NSAlertStyleInformational Txt:nil
```

This bug should be reported upstream

File: Formula/hatari.rb [hunk 1]
```diff
@@ -20,8 +16,15 @@ class Hatari < Formula
...
   resource "emutos" do
-    url "https://downloads.sourceforge.net/project/emutos/emutos/0.9.6/emutos-512k-0.9.6.zip"
-    sha256 "2c7d57cac6792d0c7e921f9655f224b039402283dd24c894b085c7b6e9a053a6"
+    url "https://downloads.sourceforge.net/project/emutos/emutos/0.9.9.1/emutos-512k-0.9.9.1.zip"
+    sha256 "ab94cd249aebd7fb1696cbd5992734042450d8b96525f707e9ad8a2283185341"
+  end
+
+  if MacOS.verion <= :high_sierra
+    patch do
+      url "url du patch"
```
Comment (fxcoudert) [inline Formula/hatari.rb]: You need to actually put the patch URL here, and its SHA-256 checksum below

Comment (btuduri) [reply Formula/hatari.rb]: I waiting to have this url. I guess, I will have it when my pull request will validate by the owner of formula-patches repository. But, I can fill up the sha256, right now.

**Comment by btuduri** [comment]
Thank you. The sierra build seems failed due to a connection timeout because the another build seems ok. I dunno how force a rebuild ? Another pull request ?

**Comment by fxcoudert** [comment]
Thanks @btuduri for the pull request!

Please, next time, do not touch the bottle block… it is replaced automatically when we merge

**Comment by btuduri** [comment]
Thank you @fxcoudert for your advices.

**Pull request closed without merging**